lib/prawn/text/formatted/parser.rb

```diff
@@ -238,6 +236,14 @@ def self.array_from_tokens(tokens)
           end
           array
         end
+
+        def self.escape(text)
+          text.gsub('&', '&amp;').gsub('>', '&gt;').gsub('<', '&lt;')
```

Comment:
What do you mean here? Like `text.gsub(/&|>|</) {|val| if val == '7' then ... }` I think that's not significantly faster and is more complex.